Styles/main.css

```diff
@@ -190,25 +191,26 @@ main {
     grid-column: 1 /span 2;
 
 }
-.links {
+#useful-links {
     grid-column: 1 / span 2; 
 }
-.gallary {
+.gallery {
     grid-column: span 2;
     width: 100%;
 }
-.gallary-thumbnails {
+.gallery-thumbnails {
     display: grid;
     grid-template-columns: 25% 25% 25% 25%;
   
 }
-.gallary img {
-    
-    
+.gallery img {
+    width: 100%;
+    display: block; 
 }
+
 .row-span-2 {
-    grid-rows: span 2;
+    grid-row: span 2;
 }
-.row-span-2.col-span-2 {
+.col-span-2 {
     grid-column: span 2;
 }
```

Comment:
The rule you have here applies to any elements that have BOTH the classes, but the lab asked for a separate rule for each of these classes.  Your 10th gallery image has both classes so each of these separate rules will apply.